specification/automation/resource-manager/readme.md

```diff
@@ -92,10 +89,43 @@ input-file:
 - Microsoft.Automation/preview/2017-05-15-preview/job.json
 ```
 
+### Tag: package-2018-01

```

Comment:
>package-2018-01 [](start = 9, length = 15)

Since this package also has files from preview versions, it should also be marked as a preview package

Reply:
Fixed